qiskit/dagcircuit/dagnode.py

```diff
@@ -100,8 +103,8 @@ def __init__(self, op, qargs=(), cargs=()):
         """Create an Instruction node"""
         super().__init__()
         self.op = op
-        self.qargs = tuple(qargs)
-        self.cargs = tuple(cargs)
+        self.qargs: Optional[Tuple[Qubit, ...]] = tuple(qargs)
+        self.cargs: Optional[Tuple[Clbit, ...]] = tuple(cargs)
```

Comment:
not an expert in type annotation. Would it be better to add them into the method declaration?

Reply:
yes, you're right; I've changed it